ts/parts/Utilities.ts

```diff
@@ -2378,7 +2378,7 @@ function animObject(
  *
  * @ignore
  */
-H.timeUnits = {
+const timeUnits = {
```

Comment:
If you do here a `const timeUnits: Highcharts.Dictionary<number> = {`, you can everywhere else get rid of the `timeUnits as any`.

Reply:
Also this https://github.com/highcharts/highcharts/blob/a8a330ff5800f81544c26228d095577c6f45d918/ts/parts/Utilities.ts#L144 has to be removed, as there is no `timeUnits` on `H` (Highcharts) set anymore.